Celeste.Mod.mm/Patches/Celeste.cs

```diff
@@ -115,6 +122,38 @@ class patch_Celeste : Celeste {
                 }
             }
 
+            // Get the splash up and running asap
+            if (!args.Contains("--disable-splash") && File.Exists("EverestSplash.dll")) {
+                Type everestSplashType = Assembly.LoadFile(Path.GetFullPath("EverestSplash.dll"))
```

Comment:
Seriously, why do jank assembly loading and reflection instead of either adding this as a dependency or just including it in the same project?

Reply:
This approach was taken because i saw the same pattern with miniinstaller and netcoreifier, no dependencies, just a dll loaded with reflection, so i thought that this was the correct solution :P
I can, of course, move to a dependency.

Reply:
Maybe @Popax21 can weigh in on why Netcoreifier works like that. The reason it needs to be a separate project at all is that both MiniInstaller and Everest need it, which is not the case for the splash screen.

Reply:
MiniInstaller is a separate project because it's a separate executable, and DiscordGameSDK is a separate project because it's distributed that way by Discord. I don't really see a justification for the splash screen to be a separate project.

Reply:
> This approach was taken because i saw the same pattern with miniinstaller and netcoreifier, no dependencies, just a dll loaded with reflection, so i thought that this was the correct solution :P I can, of course, move to a dependency.

Miniinstaller worked like that before I touched it, so I don't really know - probably a better question for jade. I assume it might have to do with conflicts between Miniinstaller / Everest's own DLLs? Either way, I agree with Kalobi on both points: this should be included right in the main Everest source, or it should at least be an ordinary dependency.

Reply:
The principal reason this made sense to me to be a separate project is for isolated testing and to ease development (faster patching because no need to run miniinstaller each time, etc.), it can definitively be just a .cs file under Celeste.mm.mod, but as something that runs in parallel to everest felt like it had to be separate. 
Most c# projects tend to have a lot of projects contributing on each other in order to create a whole app, so i saw this as just a library that everest can conditionally load.

Reply:
> > This approach was taken because i saw the same pattern with miniinstaller and netcoreifier, no dependencies, just a dll loaded with reflection, so i thought that this was the correct solution :P I can, of course, move to a dependency.
> 
> Miniinstaller worked like that before I touched it, so I don't really know - probably a better question for jade. I assume it might have to do with conflicts between Miniinstaller / Everest's own DLLs? Either way, I agree with Kalobi on both points: this should be included right in the main Everest source, or it should at least be an ordinary dependency.

I see, well then ill just throw it under `Celeste.Mod.mm`, under `Mod`? Or do i just create a new folder?

Reply:
I'd put it in Celeste.Mod.mm/Mod/Everest